andrewinsoul/MyDiary
# add homepage template

#### What does this PR do?
This PR creates the landing page for the application

#### Description of tasks completed
- Add the HTML for the index page
- Add the styling sheets for the index page
- Add scripts for the index page

#### How should this be manually tested
- Fetch the branch locally by running `git fetch origin ft-create-homepage`
- Checkout to the branch by running `git checkout ft-create-homepage`
- cd to the directory containing the HTML file and open it in your browser
- The page should look like the image below
<img width="1440" alt="Screen Shot 2019-05-07 at 7 25 43 AM" src="https://user-images.githubusercontent.com/25207671/57276400-63e88200-7099-11e9-93b5-b577f023d292.png">
